02/List.js

```diff
@@ -1,13 +1,15 @@
-import React from 'react';
+import React from 'react'
 
-import ListItem from './ListItem';
+import ListItem from './ListItem'
 
 function List(props) {
-    return (
-        <ul>
-            <ListItem />
-        </ul>
-    )
+	const { items } = props
+
+	return (
+		<ul>
+			<ListItem users={items} />
```

Comment:
Zwróć uwagę, że nazwa komponentu wskazuje, że jest to jeden element, a nie wiele. Dlatego ten komponetn powinien renderować jeden `li`. Natomiast wew. `List` powinien być `.map`, który renderuje  `ListItem` - wtedy było by to bardziej poprawne